src/commands/registry/delete.ts

```diff
@@ -0,0 +1,73 @@
+import {Flags} from '@oclif/core'
+import {TokenAwareCommand} from '../../lib/token-command'
+import {initClient} from '../../lib/client'
+import {Resource} from 'ketting'
+
+export default class Delete extends TokenAwareCommand {
+  static description = 'Deletes an event type from the registry. This only works if no events of this type have been created.'
+
+  static examples = [
+    `$ evently registry:delete -n article -e add-comment
+Deleted entity event type https://preview.evently.cloud/registry/article/add-comment
+`]
+
+
+  static flags = {
+    ...TokenAwareCommand.flags,
+    event: Flags.string({
+      char:         'e',
+      description:  'Event name',
+      required:     true,
+    }),
+    entity: Flags.string({
+      char:         'n',
+      description:  'Entity name',
+      required:     true,
+    }),
+  }
+
+  async run(): Promise<void> {
+    const {flags} = await this.parse(Delete)
+
+    const client = initClient(flags.token)
+
+    const {event, entity} = flags
+
+    const entityRes = await followByName(
+      await client.follow('registry'),
+      entity
+    )
+
+    const eventRes = await followByName(
+      entityRes,
+      event
+    )
+
+    await eventRes.delete()
```

Comment:
This delete may fail if the event type has been used in an event. See acceptance test: https://github.com/evently-cloud/api-acceptance-tests/blob/3a4c73ef04a6ef37aa3f93a272f44493efda4f50/features/registry.feature#L46

Reply:
Hey I just tested this by registing an event, appending a 'fact' event and then attempt to delete the event. Did not get an error!

Reply:
Errors do get bubbled up nicely to the user. If you use application/problem+json they'll just see the error you wrote, but I wanted to give you a screenshot to demonstrate